src/main/kotlin/traffic_simulation2/cars.kt

```diff
@@ -1,11 +1,12 @@
 package traffic_simulation2
 
 
-data  class cars ( val name : String, val drives : Boolean)
+data  class Cars ( var carName : String, var driving : Boolean)
```

Comment:
Sehe ich auch so. Außerdem benutzt man für Klassen Singular (außer in ganz seltsamen Ausnahmefällen).